pyproject.toml

```diff
@@ -1,5 +1,5 @@
 [build-system]
-requires = ["setuptools~=65.6", "wheel~=0.37.1"]
+requires = ["setuptools~=69.2.0", "wheel~=0.43.0"]
```

Comment:
This seems unrelated?

Reply:
definitely unrelated. i can revert the change.

Reply:
Please do

Reply:
done